coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/projectadministration/domain/FileEntity.java

```diff
@@ -19,4 +20,8 @@
   @Relationship(type = "RENAMED_FROM")
   @ToString.Exclude
   private List<FileEntity> oldFiles;
+
+  // TODO: this is just experimental for now
```

Comment:
Do we need this relationship? I thought we did this with a query

Reply:
We don't need it. Just forgot to delete it.